Lindalu fix description metadata (#1671)
OfficeDev/office-js-docs-pr: docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md

---
title: Requesting permissions for API use in add-ins

> description: 'Learn about different permission levels to declare in your content or task pane add-in's manifest to specify the level of JavaScript API access.'

Reviewer: Remove single quote mark in description.

```suggestion
description: 'Learn about different permission levels to declare in the manifest of a content or task pane add-in to specify the level of JavaScript API access.'
```
